tipocket-ctl/tpctl/argo.py

```diff
@@ -151,3 +152,20 @@ def gen_case_template(self):
 
     def _get_case_template_name(self):
         return self.case.meta.name
+
+
+class ArgoCronCase(ArgoCase):
+    def __init__(self, feature, case: CaseInstance, image, tidb_cluster,
+                 notify=False, notify_users=None, cron_params=None):
+        super().__init__(feature, case, image, tidb_cluster,
+                         notify, notify_users)
+        self.cron_params = cron_params
+
+    def gen_workflow(self):
+        workflow = super().gen_workflow()
+        workflow['metadata']['generateName'] += 'cron-'
```

Comment:
The name of cron workflow can be determinate.

```
workflow['metadata']['name'] = 'tpctl-{self.case.meta.name}-{self.feature}-cron'
```

Reply:
Well... Why don't we just decide that cron workflow name is ordinary workflow name + "cron-"?
If we need to change the naming pattern in the future, we would not need to change it in two places.